src/graph/graph.h

```diff
@@ -25,6 +25,17 @@
 #define GRAPH_NO_RELATION -1                 // Relations are numbered [0-N], -1 represents no relation.
 #define GRAPH_UNKNOWN_RELATION -2            // Relations are numbered [0-N], -2 represents an unknown relation.
 
+// Mask with most significat bit on 10000....
+#define MSB_MASK 1UL << ((sizeof(EntityID) * 8) - 1)
```

Comment:
It would be safer to put this whole expression in parentheses.

This seems like more work than should be necessary to set the MSB? But I can't think of a better portable solution - hopefully it compiles into a constant anyway.

Reply:
where exactly would you place the parentheses?
Agree, how can we make sure that's the case?